src/Writer.php

```diff
@@ -55,6 +57,13 @@ final class Writer
      */
     private array $ignore = [];
 
+    /**
+     * Ignores traces where the file and line matches one of the provided closures
+     *
+     * @var array<int, Closure>
+     */
+    private array $ignoreClosures = [];
```

Comment:
Can you remove this property, and make the property above to accept an `Closure|string`.